.github/workflows/release.yaml

```diff
@@ -0,0 +1,81 @@
+name: Release Helm charts
+
+on:
+  push:
+    branches:
+      - main
+    paths: 
+      - '**/Chart.yaml'
+
+jobs:
+ get-list-of-charts:
+    runs-on: ubuntu-latest
+    name: 'Get list of charts to be released'
+    outputs:
+      charts: ${{ steps.get-charts.outputs.charts }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - id: get-charts
+        name: Get list of charts to be released
+        run: |
+          # Get list of Chart.yaml files changed in the last commit
```

Comment:
Since it only checks the last commit, we should only allow squash merge right? Maybe, we should change repo settings to prevent merge commits

Reply:
Good point